lte/gateway/c/session_manager/SessionState.h

```diff
@@ -254,10 +267,14 @@ class SessionState {
 
   std::string get_session_id() const { return session_id_; }
 
-  uint32_t get_local_teid() const;
+  std::string get_subscriber_id() const;
+  uint32_t get_pdu_id() const;
 
-  void set_local_teid(uint32_t teid, SessionStateUpdateCriteria* session_uc);
+  uint32_t get_upf_local_teid() const;
```

Comment:
We should have one line between each function declarations.

Reply:
Yes, changes done.